about.html

```diff
@@ -0,0 +1,162 @@
+      The Global Summit brings together the community of experts, academics, and activist who comprise the Rocket
+      Industry network in a different country every two years.
+      This year, we hope to expand our invitation list including organization and individuals.
+    </p>
+    <p class="rocket__global_p hide desktop__show">The Global Summit brings together the community of experts, academics, and activist who comprise the Rocket
+      Industry network in a different country every two years.
+      This year, we hope to expand our invitation list including organization and individuals who want to work with
+      us on shared projects that contribute to expand the interest in rockets.
+      So if you are modelist, engineer, technician or industry fan, we hope you will consider joining us this year to build
+      a stronger community.
+    </p>
+    <div class="contact">
+      <p class="contact__p">
+        Please contact us per email for any further questions about Rocket Global Summit 2023!
+        <address>dfsalam@gmail.com</address>
+      </p>
+      
+    </div>
+  </section>
+
+  <section id="logo" class="logo__section">
+    <h2>Rocket Global Summit 2023 Logo</h2>
+    <div class="line"></div> <!--For the line -->
+    <p class="desktop__hide">The logo of Rocket Global Summit 2023 was decided through the logo competition from 8 June to 7 July.</p>
+    <p class="hide desktop__show">The logo of Rocket Global Summit 2023 was decided through the logo competition from 8 June to 7 July.
+      After online public voting and Selection Comittee's judging process, the log from Naresh Agrawal won the prize.
+      After consultation with Mr. Agrawal we optimized the prize-winning-logo for the next year´s summit as following:
+    </p>
+    <img src="./images/big_logo.png" alt="logo">
+  </section>
+  <section id="past_summits" class="past__conferences">
+    <h2>See the past Rocket Global Summits</h2>
+    <div class="line"></div>
+    <p>Take a look at the last two Rocket Global Summits which took place in Buenos Aires and Warsaw.</p>
+    <div class="past__summits">
+      <div class="past__conferences__card1">
+        <h1 class="past__year">2021</h1>
+        <h3 class="past__place">Rocket Global Summit 2021 in Buenos Aires</h3>
+      </div>
+      <div class="past__conferences__card2">
+        <h1 class="past__year">2019</h1>
+        <h3 class="past__place">Rocket Global Summit 2021 in Warsaw</h3>
+      </div>
+    </div>
+  </section>
+  <section id="partner" class="partner desktop__hide">
+    <h1>Partner</h1>
+    <div class="line"></div>
+  </section>
+  <footer>
+    <img src="./images/big_logo.png" alt="Logo">
+    <p>Original design idea by <a href="https://www.behance.net/adagio07" target="_blank" rel="noopener">Cindy Shin in Behance</a>; according to the <a href="https://creativecommons.org/licenses/by-nc/4.0/">Creative Commons license of the design</a></p>
+  </footer>
```

Comment:
- [ ] Good job so far But I noticed that your about page footer doesn't follow the original design spacing(padding/margin) and it is a bit different from it. Please make sure to stick to the original design as much as possible :+1: 

#### Your design
![Screenshot_20230107_235725(1)](https://user-images.githubusercontent.com/6615394/211174687-974678df-3c87-46cb-b5e9-76aa44cba368.jpg)


#### This is just an example of an index footer from the Original design 
![Screenshot_20230107_235345(1)](https://user-images.githubusercontent.com/6615394/211174579-052f9503-16f7-4130-941f-c44531cc6991.jpg)

#### Note the about page footer background shoud be  Dark

Reply:
![image](https://user-images.githubusercontent.com/93690227/211181042-a0714805-e895-459a-8ab2-63b548c05d28.png)
I totally agree with the distribution inside the footer, but the original background design is dark, as the picture shows.